src/utils/date.ts

```diff
@@ -198,17 +217,23 @@ export function isRelativeButNotNow(
   return !(diff / 60 >= minuteThreshold || diff / 60 <= 1);
 }
 
-export function formatLocaleTime(date: Date | string, language: Language) {
-  const lang = language ?? DEFAULT_LANGUAGE;
-  switch (lang) {
-    case Language.Norwegian:
-      return format(parseIfNeeded(date), 'HH:mm');
-    case Language.English:
-      return format(parseIfNeeded(date), 'HH:mm');
-    case Language.Nynorsk:
-      return format(parseIfNeeded(date), 'HH:mm');
-  }
+type LocalTimeFormatOptions = {
+  ignoreTimeZone: boolean;
+};
+export function formatLocaleTime(
+  date: Date | string,
+  // @TODO: No longer in use. deprecate and remove
+  _language: Language,
+  options: Partial<LocalTimeFormatOptions> = {ignoreTimeZone: false},
+) {
+  const parsed = parseIfNeeded(date);
+  const opts = {
+    ignoreTimeZone: false,
+    ...options,
+  };
+  return format(parsed, 'HH:mm', opts);
```

Comment:
This used language at some point to probably support different formats, but now it seems to only do 24-hour. Haven't updated all references but removed unneeded code